.github/workflows/release_parallel.yml

```diff
@@ -0,0 +1,218 @@
+name: Parallel Release
+on:
+    workflow_call:
+        secrets:
+            GPG_SIGNING_KEY:
+                required: true
+            GH_TOKEN:
+                required: true
+            NPM_TOKEN:
+                required: true
+            LASTMJS_GITHUB_TOKEN:
+                required: true
+
+jobs:
+    prepare-release:
+        name: Prepare Release
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }} # All commits must be verified
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        outputs:
+            release-version: ${{ steps.get-version.outputs.release-version }}
+            test-infos: ${{ steps.get-test-infos.outputs.test-infos }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.ref || github.ref }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN || github.token }}
+
+            - id: get-version
+              run: |
+                  BRANCH_NAME="${{ github.event.pull_request.head.ref || github.ref_name }}"
+                  RELEASE_VERSION="${BRANCH_NAME:9}"
+                  echo "release-version=$RELEASE_VERSION" >> $GITHUB_OUTPUT
+
+            - id: get-node-version
+              uses: ./.github/actions/get_node_version
+
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ steps.get-node-version.outputs.node-version }}
+                  registry-url: https://registry.npmjs.org
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+            - name: Install curl
+              run: sudo apt-get install curl -y
+
+            - id: get-dfx-version
+              uses: ./.github/actions/get_dfx_version
+
+            - name: Install dfx
```

Comment:
I feel like we're doing this in a couple places now...could this be made reusable somehow?

Reply:
it is literally already pulled out to be reusable, the only thing this job does is call a script

Reply:
We could thy echoing the path inside the script

Reply:
done